src/Input.js

```diff
@@ -193,7 +193,8 @@ Input.defaultProps = {
  */
 Input.propTypes = {
     name: propTypes.string.isRequired,
-    type: propTypes.string.isRequired,
+    type: propTypes.oneOf('text', 'number', 'password', 'email', 'url', 'tel')
```

Comment:
According to the specs, this needs to be an array, Does it work this way?
https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Regardless of whether it works or not, I'd stick to the official way

Reply:
Sorry, that was just plain sloppy. Corrected it now